lib/screens/dashboards_screen.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:cogboardmobileapp/providers/dashboards_provider.dart';
+import 'package:cogboardmobileapp/screens/settings_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class DashboardsScreen extends StatelessWidget {
+  static const routeName = '/dashboards';
+
+  @override
+  Widget build(BuildContext context) {
+    final dashboardsProvider = Provider.of<DashboardsProvider>(context);
+    final dashboardTabs = dashboardsProvider.dashboardTabs;
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(dashboardTabs[dashboardsProvider.dashboardTabIndex].title),
+        actions: <Widget>[
+          IconButton(
+            icon: Icon(Icons.settings),
+            color: Theme.of(context).accentColor,
+            onPressed: () {
+              Navigator.of(context).pushNamed(SettingsScreen.routeName);
+            },
+          )
+        ],
+      ),
+      body: dashboardTabs[dashboardsProvider.dashboardTabIndex].page,
+      backgroundColor: Theme.of(context).primaryColor,
+      bottomNavigationBar: BottomNavigationBar(
+        onTap: (index) => dashboardsProvider.setDashboardTabIndex(index),
+        selectedItemColor: dashboardTabs[dashboardsProvider.dashboardTabIndex]
+            .selectedTabColor,
+        unselectedItemColor: Theme.of(context).accentColor,
+        currentIndex: dashboardsProvider.dashboardTabIndex,
+        items: [
+          BottomNavigationBarItem(
+            backgroundColor: Theme.of(context).primaryColor,
+            icon: Icon(Icons.star),
+            title: Text('Favourite'),
```

Comment:
I would recommend to keep all strings in one file (example here: https://stackoverflow.com/a/53653788) to easily maintain all of them, prevent defining duplicated strings and to support various languages

Reply:
👍

Reply:
Please implement this for all Labels. Let the names of the variables describe where they are used if possible.